tests/dialects/test_mysql.py

```diff
@@ -10,7 +10,19 @@ def test_ddl(self):
         self.validate_identity("UPDATE items SET items.price = 0 WHERE items.id >= 5 LIMIT 10")
         self.validate_identity("DELETE FROM t WHERE a <= 10 LIMIT 10")
         self.validate_identity(
-            "INSERT INTO x VALUES (1, 'a', 2.0) ON DUPLICATE KEY UPDATE SET x.id = 1"
```

Comment:
This test was added in https://github.com/tobymao/sqlglot/commit/5df76cee6 and is invalid in MySQL (tested locally with MySQL version 8). This also checks out with the examples in their documentation page for `INSERT ... SELECT`.